manager/src/main/java/com/redhat/service/bridge/manager/models/Processor.java

```diff
@@ -168,4 +187,15 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id);
     }
+
+    private Set<com.redhat.service.bridge.infra.models.filters.Filter> buildFilters() {
```

Comment:
I would move this logic to `ProcessorDAO` to keep the entity ~without logic. What do you think?

Reply:
@danielezonca wdym? This `ProcessorEntity` contains only `FilterEntity`s. 
We use this `buildFilters` method only in the `toDTO` and `toResponse` methods so to convert the `FilterEntity` to `Filter`.  If `ProcessorDAO` should return `ProcessEntity` instances which contain `FilterEntity`, not `Filter.